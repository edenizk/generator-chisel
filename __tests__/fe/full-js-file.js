const { defaultAnswers } = require('./helpers');

describe('Full file js', () => {
  test('Full file js', async () => {
    await global.chiselTestHelpers.generateProjectWithAnswers(
      ['create'],
      defaultAnswers,
    );

    await global.chiselTestHelpers.fileMatchesSnapshot('./dist/scripts/app.b6c8c19f.full.js');
  });
});
