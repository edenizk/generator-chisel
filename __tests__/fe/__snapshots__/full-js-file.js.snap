// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Full file js Full file js 1`] = `
"/******/ (() => { // webpackBootstrap
/******/ 	\\"use strict\\";
/******/ 	/* webpack/runtime/compat */
/******/ 	
/******/ 	
/******/ 	// Chisel: Allow public path to be modified during runtime
/******/ 	__webpack_require__.p = (typeof document !== \\"undefined\\" && document.documentElement.dataset.webpackPublicPath) || __webpack_require__.p;
/******/ 	
/************************************************************************/
var __webpack_exports__ = {};

;// CONCATENATED MODULE: ./src/scripts/modules/greeting.js
const greeting = name => {
  const element = document.querySelector('.js-greeting');

  if (element) {
    element.innerHTML = name;
  }
};

/* harmony default export */ const modules_greeting = (greeting);
;// CONCATENATED MODULE: ./src/scripts/app.js
/*
  Project: FrontEnd
  Author: Xfive Tester
 */

modules_greeting('World');
/******/ })()
;"
`;
